ansibleplaybookgrapher/parser.py

```diff
@@ -160,6 +162,7 @@ def __init__(
         self.include_role_tasks = include_role_tasks
         self.playbook_path = playbook_path
         self.playbook_name = playbook_name
+        self.exclude_roles = exclude_roles
```

Comment:
You can default to an empty list: 
```suggestion
        self.exclude_roles = exclude_roles or []
```

As such, no need to check if the list is None later on.